sales_tax.rb

```diff
@@ -0,0 +1,48 @@
+class Product
+    attr_accessor :name , :price , :quantity , :is_imported , :is_exampt
+    def initialize(name,price,quantity,is_imported,is_exampt)
+        @name = name
+        @price = price
+        @quantity = quantity
+        @is_imported = is_imported
+        @is_exampt = is_exampt
+    end
+end
+
+$products = []
```

Comment:
@mohseengs these all should be wrapped around a class no loose variables or methods should be present.